### ISSUE TITLE
Interface MTU

### ISSUE THREAD

**Issue by wazoo** [aruba/aoscx-ansible-collection#38]
I can't seem to figure out how to set interface MTU with this collection, is that missing?  

If so, is that because its missing from pyaoscx as well?  I want to open a feature request in the right place so if that is the right place I will do that otherwise please consider this a feature request here.

**Comment by tchiapuziowong** [comment]
Hi @wazoo that's correct it seems this option is missing from the `aoscx_interface` module - I'll open this with our development team and update this issue with a status on planned development/publishing - thank you for bringing this to our attention!

**Comment by wazoo** [comment]
Well I am glad I didn't just miss it!  That would be awesome, i'll keep an eye out.  I also noticed that LAG settings are missing as well unless I missed those, I can open another issue if you don't want to lump them together in this ticket.

**Comment by tchiapuziowong** [comment]
We're currently working on development of a lag interface module so we'll be providing announcements once that is complete 👍

**Comment by alagoutte** [comment]
@kchavesr can be nice to add also IP MTU ?

**Comment by grid-dev** [comment]
> We're currently working on development of a lag interface module so we'll be providing announcements once that is complete 👍

Any updates on the lag interface? We would need this urgent and am not sure if we should go another route.

**Comment by tchiapuziowong** [comment]
@grid-dev lag interface is still in testing/validation with our internal teams but will be a part of the next release - in the interim you can use our SSH module `aoscx_config` to configure LAG features using CLI commands : https://developer.arubanetworks.com/aruba-aoscx/docs/using-the-aos-cx-ansible-collection#sshcli-modules
